Lib/test/test_syntax.py

```diff
@@ -2852,6 +2852,11 @@ def test_assign_del(self):
         # report "cannot delete name"
         self._check_error("del a += b", "invalid syntax")
 
+    def test_parenthesized_named_expression_target(self):
+        self._check_error(
+            "((a) := 1)",
+            "cannot parenthesize target name in assignment expression")
+
```

Comment:
I think it'd be best if this went in the docstring at the top of the file instead of a separate test. It can probably go at the very bottom of that docstring with a short description of the error being tested, something along the lines of `Invalid parenthesization inside assignment expressions`.